docs/io/optional/logging_configuration.ipynb

```diff
@@ -0,0 +1,2226 @@
+{
```

Comment:
<p>Change "an User" to "a user".</p><p> </p><p>Remove "the arguemnts that is", so it reads as "If a user specifies both the parameters...".</p><p> </p><p>Remove 'then' after the comma.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>